packages/web/src/apollo/index.tsx

```diff
@@ -188,12 +194,32 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
     httpLink = new HttpLink({ uri, fetch: crossFetch, ...httpLinkConfig })
   }
 
-  // The order here is important. The last link *must* be a terminating link like HttpLink.
+  // Our terminating link needs to be smart enough to handle subscriptions, and if the GraphQL query
+  // is subscription it needs to use the SSELink (server sent events link).
+  const terminatingLink = apolloClient.split(
+    ({ query }) => {
+      const definition = getMainDefinition(query)
+
+      return (
+        definition.kind === 'OperationDefinition' &&
+        definition.operation === 'subscription'
+      )
+    },
+    new SSELink({
+      url: uri,
```

Comment:
If in single connection mode, then the endpoint needs to end in `/graphql/stream` s this is a todo for future